tests/app/sftp/test_ftp_client.py

```diff
@@ -21,7 +20,6 @@ class SendZipMocks:
     yield SendZipMocks
 
 
-@freeze_time('2016-01-01T17:00:00')
```

Comment:
Is time freeze not needed anymore because we changed the tests in a way that doesn't rely on the date, where they previously did, but forgot to remove the time freezes?

Reply:
Yes, that's right. We used to generate the DVLA filename in this repo, so the tests contained filenames with datetimes in them and we needed freeze_time for that. But we removed that behaviour in https://github.com/alphagov/notifications-ftp/pull/229/commits/32d8dba47b21361834f2594a28f207c41a619c24 - you can see a couple of examples of how the tests used to look there.